spec/lang/types.md

```diff
@@ -142,7 +142,8 @@ impl Type {
         match self {
             Int(int_type) => int_type.size,
             Bool => Size::from_bytes_const(1),
-            Ptr(_) => T::PTR_SIZE, // FIXME: sometimes 2 words
+            Ptr(p) if p.meta_kind() == PointerMetaKind::None => T::PTR_SIZE,
+            Ptr(_) => libspecr::Int::from_u64(2) * T::PTR_SIZE,
```

Comment:
```suggestion
            Ptr(_) => Int::from(2) * T::PTR_SIZE,
```

Reply:
Doesn‘t work, `Int` is importet as `use Type::*`. Could do `Ptrsize + Ptrsize` tho